app/routes/subtasks.js

```diff
@@ -0,0 +1,18 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import RSVP from 'rsvp';
```

Comment:
```suggestion
import RSVP from 'rsvp';
import Route from '@ember/routing/route';

import { inject as service } from '@ember/service';
```

ordering for readability